plugins/source/aws/resources/services/docdb/clusters_fetch.go

```diff
@@ -0,0 +1,40 @@
+package docdb
+
+import (
+	"context"
+
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/service/docdb"
+	"github.com/aws/aws-sdk-go-v2/service/docdb/types"
+	"github.com/cloudquery/cloudquery/plugins/source/aws/client"
+	"github.com/cloudquery/plugin-sdk/schema"
+)
+
+func fetchDocdbClusters(ctx context.Context, meta schema.ClientMeta, _ *schema.Resource, res chan<- interface{}) error {
+	c := meta.(*client.Client)
+	svc := c.Services().DocDB
+
+	input := &docdb.DescribeDBClustersInput{}
+
+	for {
```

Comment:
Suggestion: DBClusters supports a paginator, which should simplify our code a little since we wouldn't need to handle the markers:  https://docs.aws.amazon.com/sdk-for-go/api/service/docdb/#DocDB.DescribeDBClustersPages